.travis/run_tests.sh

```diff
@@ -4,7 +4,7 @@ if [[ $TRAVIS ]]; then
     cd "$TRAVIS_BUILD_DIR" || exit 2
 
     # Check modified PHP files with PHP's internal syntax checker
-    git diff --name-only HEAD^ | grep '\.php$' | xargs -r -n 1 php -l || exit 1
+    git diff --name-only --diff-filter=ACMRTUXB HEAD^ | grep '\.php$' | xargs -r -n 1 php -l || exit 1
```

Comment:
Ah, so it was checking _all_ modified files, including deleted ones? This change filters for everything other than Deleted (`D`)?